Octo-Bot/packet-generator-bot/bot/utils/TimeUtils.py

```diff
@@ -0,0 +1,25 @@
+
+MIN_IN_SECS = 60
+HOUR_IN_SECS = 3600
+DAY_IN_SECS = 86400
+
```

Comment:
Can you a little documentation for this function?
Including expected inputs/arguments.

Reply:
Added!